GOVERNANCE_DOC.md

```diff
@@ -0,0 +1,619 @@
+
+`Urgent` proposals also work with the aforementioned threshold, except there is another condition that can accelerate the acceptance of the proposal. Namely, if the ratio of `Yes` votes to `InitTotalVotingPower` exceeds 2/3, `UrgentProposal` will be immediately accepted, even if the `Voting period` is not finished. `InitTotalVotingPower` is the total voting power of all bonded Atom holders at the moment when the vote opens.
+
+### Inheritance
+
+If a delegator does not vote, it will inherit its validator vote.
+
+- If the delegator votes before its validator, it will not inherit from the validator's vote.
+- If the delegator votes after its validaotor, it will override its validator vote with its own vote. If the proposal is a `Urgent` proposal, it is possible that the vote will close before delegators have a chance to react and override their validator's vote. This is not a problem, as `Urgent` proposals require more than 2/3rd of the total voting power to pass before the end of the voting period. If more than 2/3rd of validators collude, they can censor the votes of delegators anyway. This is an inherent limit of byzantine fault tolerant systems. 
+
+### Validator’s punishment for non-voting
+
+Validators are required to vote on all proposals to ensure that results have legitimacy. Voting is part of validators' directives and failure to do it will result in a penalty. 
+
+If a validator’s address is not in the list of addresses that voted on a proposal and if the vote is closed (i.e. `MinDeposit` was reached and `Voting period` is over), then any user of the Hub can submit a `TxGovernancePenalty` that will result in a partial slash `GovernancePenalty` of the validator's stake. The user that submitted the `TxGovernancePenalty` transaction will receive `GovernanceReward` from the `GovernancePenalty`.
+
+*Note: Need to define values for `GovernancePenalty` and `GovernanceReward`*
+
+**Exception:** If a proposal is a `Urgent` proposal and is accepted via the special condition of having a ratio of `Yes` votes to `InitTotalVotingPower` that exceeds 2/3, validators cannot be punished for not having voted on it. That is because the proposal will close as soon as the ratio exceeds 2/3, making it mechanically impossible for some validators to vote on it.
+
+### Governance key and governance address
+
+Validators can make use of an additional slot where they can designate a `Governance PubKey`. By default, a validator's `Governance PubKey` will be the same as its main PubKey. Validators can change this `Governance PubKey` by sending a `Change Governance PubKey` transaction signed by their main `Consensus PubKey`. From there, they will be able to sign vote using the `Governance PrivKey` associated with their `Governance PubKey`. The `Governance PubKey` can be changed at any moment.
```

Comment:
We should enforce for these two keys to be different. Otherwise the default is a security weakness. It's like leaving password as `admin` on your router.

Reply:
Hmm can you give an explicit example where this would cause a security issue?

Reply:
In the default case. The validation key now needs to be available to also sign votes. This means it is easier duplicated or the validator needs to have hot access to the HSM. 

Most likely the key will be duplicated. A validator should never have the validation key in a hot HSM to sign proposals.

Reply:
Ok then why not force the validator to specify a `Governance PubKey` in its `TxBond`, meaning there is no default choice? If the validator then inputs the same PubKey as his `Consensus PubKey`, it's its problem...

Reply:
Yeah I can build this logic into the staking module... I guess there is some overlap, there should only really be one candidate object but it should support both staking and governance

Reply:
Probably it should be "signed by their main `Consensus PrivKey` instead of `Consensus PubKey`.